spec/index.bs

```diff
@@ -2259,6 +2247,9 @@ A {{GPUBindGroup}} object has the following internal slots:
                 with {{GPUBufferUsage/STORAGE}} flag.
             1. The bound part designated by |bufferBinding|.{{GPUBufferBinding/offset}} and
                 |bufferBinding|.{{GPUBufferBinding/size}} must reside inside the buffer.
+            1. The effective binding size, that is either explict in |bufferBinding|.{{GPUBufferBinding/size}}
```

Comment:
Eventually it would be nice if this "effective binding size" pointed at a real variable instead of being described vaguely like this.